Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM docker:dind
 
 ARG NETWORK=prod
-ARG BRANCH=main
+ARG BRANCH=as/docker-internal-hosts
```

Comment:
i had to remove the hosts mapping from server, otherwise it wouldn't talk to host.docker.internal correctly

Reply:
https://github.com/AudiusProject/audius-docker-compose/pull/404